runtime-sdk/modules/rofl-market/src/lib.rs

```diff
@@ -501,22 +521,32 @@ impl<Cfg: Config> Module<Cfg> {
         Ok(())
     }
 
-    #[handler(call = "roflmarket.InstanceUpdateMetadata")]
-    fn tx_instance_update_metadata<C: Context>(
-        ctx: &C,
-        body: types::InstanceUpdateMetadata,
-    ) -> Result<(), Error> {
-        <C::Runtime as Runtime>::Core::use_tx_gas(Cfg::GAS_COST_CALL_INSTANCE_UPDATE_METADATA)?;
+    #[handler(call = "roflmarket.InstanceUpdate")]
```

Comment:
Is there a reason why some methods (such as this one, and the removed `"roflmarket.InstanceUpdateMetadata"`) are missing from the go client? 
Having a complete client in Go with all methods available from the runtime module might be nice.

Reply:
Oh, it is just because they can only be called by the scheduler and I'm lazy :-) We can add them for completeness, but unless the Go client is used to implement a scheduler (which is entirely possible given TDX), users would never be able to call them.